README.md

```diff
@@ -21,6 +21,28 @@ A plugin that displays user-friendly messages when Video.js encounters an error.
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 ## Getting Started
+**Importing via NPM/Babel/Browserify**
+```shell
+npm install videojs-errors
+```
+
+You may need to add videojs-errors as plugin to the videojs module:
+```js
+import videojs from 'video.js';
+import videojsErrors from 'videojs-errors';
+
+//For VideoJS 5 and below
+videojs.plugin('errors', videojsErrors);
+
+//For VideoJS 6
+videojs.registerPlugin('errors', videojsErrors);
```

Comment:
Yep, also saw that. But it seems to get deregistered when importing via Babel.

```js
import videojs from "video.js";
import videojsErrors from "videojs-errors";

//videojs.registerPlugin('errors', videojsErrors);

const player = videojs('#test', {
  controls: true,
  autoplay: false,
  aspectRatio: '16:9'
});

console.log(player.errors); //undefined
```

Btw sorry, I think this should be an issue on its own rather than reinitializing it.